.github/workflows/pull_request_label.yml

```diff
@@ -11,8 +11,10 @@ jobs:
         timeout-minutes: 1
         steps:
             - name: Checkout repository
+              if: ${{ !env.ACT }}
```

Comment:
This isn't the right place for this. I was expecting to put it into the `action.yml` of the semver label action